templates/docs/examples/templates/typographic-spacing.html

```diff
@@ -3,1684 +3,232 @@
 
 {% block style %}
 <style>
-  .flex-container {
-    display: flex;
-    justify-content: space-between;
-  }
-  .row {
-    max-width: 100vw;
-  }
+.comment,
+.prolog,
+.doctype,
+.cdata {
+	color: #666;
+}
+
+.punctuation {
+	color: #757575;
+}
+
+.namespace {
+	opacity: .7;
+}
+
+.property,
+.tag,
+.boolean,
+.number,
+.constant,
+.symbol,
+.deleted {
+	color: #0e8620;
+}
+
+.selector,
+.attr-name,
+.string,
+.char,
+.builtin,
+.inserted {
+	color: #860048;
+}
+
+.operator,
+.entity,
+.url,
+.language-css .string,
+.style .string {
+	color: #9e5a06;
```

Comment:
Any reason for having `.language-css .string` coloured different than `.string` (above selector)? Also what is `.style .string` for? `.style` doesn't seem to be prism related class name (at least as far as I see in default theme).